cmd/cni-metrics-helper/metrics/metrics.go

```diff
@@ -397,13 +401,13 @@ func metricsListGrabAggregateConvert(t metricsTarget) (map[string]*dto.MetricFam
 	resetMetrics(interestingMetrics)
 
 	targetList := t.getTargetList()
-	glog.Info("targetList: ", targetList)
-	glog.Info("len(targetList)", len(targetList))
+	log.Infof("targetList: %v ", targetList)
+	log.Infof("len(targetList) %d", len(targetList))
 	for _, target := range targetList {
-		glog.Infof("Grab/Aggregate metrics from %v", target)
+		log.Infof("Grab/Aggregate metrics from %v", target)
 		rawOutput, err := t.grabMetricsFromTarget(target)
 		if err != nil {
-			glog.Errorf("Failed to getMetricsFromTarget: %v", err)
+			log.Warnf("Failed to getMetricsFromTarget: %v", err)
```

Comment:
Warnings should be for things that should be addressed by an operator/admin. Is the above really something that should be addressed by an admin? Recommend using `Infof` instead.